contentcuration/contentcuration/frontend/settings/views/Storage/RequestForm.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  
+  <form @submit.prevent="submit">
+  </form>
+
+</template>
+
+
+<script>
+  const formElements = {
+    "Nature of your content": [
+      {
+        key: 'storage_amount',
+        placeholder: 'Storage amount requested (e.g. 10GB)',
+        label: '',
+      },
+      {
+        key: 'num_resource',
+        placeholder: 'Number of resources',
+        label: 'Approximately how many individual resources are you planning to upload?',
+      },
+      {
+        key: 'avg_size',
+        placeholder: 'Size',
+        label: 'Average size of each resource',
+      },
+      {
+        key: 'type_of_content',
+        placeholder: 'Type of content',
+        label: 'What kind of content are you uploading? Please specify',
+      },
+      {
+        key: 'author_curator_aggregator',
+        placeholder: 'Response',
+        label: 'Who is the author (creator), curator (organizer), and/or aggregator (maintainer) of your content? Please specify',
+      },
+      {
+        key: 'sample_link',
+        placeholder: 'Paste link here',
+        label: 'Please provide a link to a sample of your content (on Studio or from source site)',
+      },
+    ],
+    "Who can use your content?": [
+      {
+        key: 'license',
+        checkbox_options: [
+          "CC BY",
```

Comment:
Note: can use [ConstantsTranslationMixin](https://github.com/learningequality/studio/blob/vue-refactor/contentcuration/contentcuration/frontend/shared/mixins.js#L164) to translate these in the form